tests/nextflow.config

```diff
@@ -4,3 +4,4 @@
 ========================================================================================
 */
 
+trace.overwrite = true
```

Comment:
do you really need that?

Reply:
@kbestak This was added in this commit? Any idea if it's required?
https://github.com/nf-core/molkart/commit/159005967c74ef825f0fa99624a3df10810f0278

Reply:
![image](https://github.com/user-attachments/assets/15042c1f-8c5a-4337-bdd9-f218b5c1a1e4)